packages/icon/test/icon.test.ts

```diff
@@ -69,6 +71,28 @@ describe('Icon', () => {
         await expect(el).to.be.accessible();
     });
 
+    it('loads w/ invalid src, error dispatching', async () => {
+        const error = Sinon.spy();
+        const el = await fixture<Icon>(
+            html`
+                <sp-icon
+                    label="Image Icon"
+                    src="invalid-image-src"
+                    @error=${error}
+                ></sp-icon>
+            `
+        );
+
+        await elementUpdated(el);
+
+        await expect(el).to.be.accessible();
+
+        // Skipping the test case expectation for webkit because of error event not dispatching bug for the same, https://github.com/microsoft/playwright/issues/22332
+        if (!isWebKit()) {
+            expect(error).to.be.calledOnce;
```

Comment:
[This is why we wait for CI to run before merging](https://app.circleci.com/pipelines/github/adobe/spectrum-web-components/15630/workflows/576533f7-9d4d-428b-80a2-6c0c0ac71011/jobs/322293)... It looks like the timing on this test can be flaky (at least in Firefox). Hopefully, it's just timing an not something more complicated. Can you try this?
```suggestion
            await waitUntil(() => error.calledOnce, 'The error was thrown.');
```
This will give all of the browsers as much as two seconds for the error to be thrown. It _should_ be OK, as in some passes Firefox has passed, but it's hard to tell 100% what's going on from those logs.

Reply:
You need to import this method.